### ISSUE TITLE
[Legitimate Site Blocked] markmeta.finance

### ISSUE THREAD

**Issue by MarkMeta** [MetaMask/eth-phishing-detect#6517]
https://markmeta.finance/

**Comment by devenblake** [comment]
Duplicate of #6516.

**Comment by MarkMeta** [comment]
MarkMeta is an upcoming metaverse project which is in development. Kindly review it again and unblock the site.
https://markmeta.finance/
